### ISSUE TITLE
Creation of "Undetermined Mode of inheritance" ontology

### ISSUE THREAD

**Issue by courtneythaxton** [obophenotype/human-phenotype-ontology#4238]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
"Undetermined mode of inheritance"

### 2. Synonyms
unknown mode of inheritance
unknown inheritance

We would like to create a new HPO term for when an inheritance is undetermined. This is part of the ClinGen gene curation efforts, and is usually assigned to gene-disease relationships that are more newly associated, or disputed or refuted. Due to our curation workflow, we require an HPO for the mode of inheritance to create a gene-disease record. 

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

A mode of inheritance that is not easily discernible, or is questionable.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP:0000005

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
ClinGen:courtneythaxton @courtneythaxton
ClinGen:erinriggs @erinriggs
ClinGen:mattwright @mwright
ClinGen jimmyzqhen @jimmyzhen
ClinGen:jennygoldstein @jennygoldstein
ClinGen:marinadistefano @marinadistefano
ClinGen:tristannelson @tnavatar
ClinGen:scottgoehringer @sgoehringer

Also looping in @nicolevasilevsky as she has helped with ontologies for ClinGen.

**Comment by pnrobinson** [comment]
Dear ClinGen colleagues -- there are a number of reasons why it is not a good idea to have an ontology term for "Unknown mode of X", and we could gladly discuss this on a call if you like. Would it be a viable option for you just to use the parent term (Mode of inheritance: https://hpo.jax.org/app/browse/term/HP:0000005). The interpretation of this would be that there is a mode of inheritance (i.e., genes play "some" role), but that no further commitment to a particular kind of mode of inheritance is made. 

The other suggestion (semidominant) is fine, and we will add that term.

**Comment by courtneythaxton** [comment]
Dear Peter,

Thank you for the prompt reply and great suggestion. We have discussed your suggestion with several colleagues, and have agreed to move forward with the use of the general Mode of inheritance HPO term (HP:0000005) in cases where the mode of inheritance may not be determined at the time of the curation effort. Please let me know if I need to further close out or delete this ticket.

Thank you for your consideration and addition of the semi dominant term, as it will be a valuable addition to our curation efforts.

Best wishes,
Courtney

Courtney Thaxton, Ph.D.
ClinGen Senior Biocurator
Software Design Specialist
Berg Lab, UNC
Dept. of Genetics
120 Mason Farm Road
5100 B, Genetic Medicine Building
CB#7264
University of North Carolina,
Chapel Hill, NC 27599
919-966-9562

**Comment by pnrobinson** [comment]
-- closing this issue. Just added the semidominant term as well.
